.github/workflows/test-environment.yml

```diff
@@ -24,15 +32,18 @@ on:
         required: true
         description: "Elastic Cloud deployment region"
         default: "gcp-us-west2"
+        options:
+          - gcp-us-west2
+          - aws-eu-west-1
         type: string
```

Comment:
Since the field is of `type: string`, using `options` on this field is obsolete (you still need to fill the text box). To enable the use of selection, it is necessary to update it to `type: choice`. However, I'm not sure which is better. On one hand, since we are using specific options for the region, it might be preferable to provide selection. On the other hand, if we need to extend to other environments, this will require more options. Providing `type: string` at least allows deploying the environment to known regions without updating the workflow.